src/ApiCalls.ts

```diff
@@ -67,6 +67,7 @@ export async function sendFavoriteCamps(
       })
     );
 
+
     console.log("Server responses:", responses); // Log the server responses
```

Comment:
If you like console.logs that totally fine - but sometimes I've seen people use console.error for situations just like this. Feel free to try it for fun